uqcsbot/manage_cogs.py

```diff
@@ -36,6 +43,24 @@ async def manage_cogs(
                     await self.bot.unload_extension(f"uqcsbot.{cog}")
                 case "reload":
                     await self.bot.reload_extension(f"uqcsbot.{cog}")
+                case "load-all":
+                    if (cogs := COG_GROUPS.get(cog)) is not None:
+                        for x in cogs:
+                            await self.bot.load_extension(f"uqcsbot.{x}")
+                    else:
+                        raise RuntimeError(f"Unknown cog group {cog}")
+                case "unload-all":
+                    if (cogs := COG_GROUPS.get(cog)) is not None:
+                        for x in cogs:
+                            await self.bot.unload_extension(f"uqcsbot.{x}")
+                    else:
+                        raise RuntimeError(f"Unknown cog group {cog}")
+                case "load-all":
```

Comment:
Should this be `case "reload-all"`?

Reply:
yeah ty, I'm midway through revamping my entire dev workflow so everything - font included - is *slightly* to the left of what I expect...